docs/release_notes/plugin_api/0.1.x.rst

```diff
@@ -0,0 +1,8 @@
+Plugin API 0.1.alpha Release Notes
```

Comment:
Yeah, since we're going with x.y.z I'll rename the first to `0.0.x.rst` (or should it be `0.0.z.rst`?).
I don't think it should be `0.0.1.alpha.rst` because we want to group all the z releases together